.github/workflows/django-multitenant-tests.yml

```diff
@@ -0,0 +1,80 @@
+on:
+  push:
+    branches:
+      - "**"
+  workflow_dispatch:
+jobs:
+  static-checks:
```

Comment:
There are mainly two jobs.
1. Static code analysis and format check
2. Test jobs

Reply:
🎉 love that we have this variety of tests